src/Raven.Studio/wwwroot/App/views/manage/adminLogs.html

```diff
@@ -68,8 +68,8 @@ <h4>Log filters</h4>
                     </div>
                 </div>
             </div>
-            <input class="form-control margin-right margin-right-sm" placeholder="Filter messages" data-bind="textInput: filter">
-            <div class="checkbox no-padding-left">
+            <input class="form-control margin-right margin-right-sm" placeholder="Filter server messages" data-bind="textInput: filter" title="Filter server messages. Studio messages will still show.">
```

Comment:
I don't think we want that - no one except us won't know what's server or studio message.

Reply:
ok - removed titles